package.json

```diff
@@ -1,29 +1,43 @@
 {
   "name": "vui-button",
-  "description": "LESS mixins and CSS for applying button styles",
-  "version": "0.5.3",
-  "author": "D2L Corporation",
-  "license": "Apache-2.0",
+  "version": "0.5.4",
```

Comment:
Suggest maybe bumping to 0.6.0 since this is technically new non-breaking features.